about/licenses/index.md

```diff
@@ -69,6 +69,124 @@ If your repository was approved for release under a different open source
 license, the text of that license will be provided by the Innovation and
 Partnerships Office.
 
+If your project has more than one license, then you must include *all* of
+the relevant licenses in your repository.  We recommend that you name
+them with descriptive suffixes.  For example, if your project is dual
+licensed under Apache-2.0 and MIT, you should have top-level
+`LICENSE-APACHE` and `LICENSE-MIT` files. Because GitHub does not
+automatically detect multiple licenses, we recommend *also* adding a
+top-level `COPYRIGHT` file with a summary of license details. GitHub's
+"view license" link will then point to this file.  You can look at
+[Spack](https://github.com/spack/spack) and its `README.md`, `COPYRIGHT`,
+and `LICENSE-*` files for an example of how to organize a project with
+two licenses.
+
+### Other considerations
+
+In addition to the required files above, you should read the following
+sections and determine whether they apply to your code.
+
+#### Contribution policies
+
+If you host your code publicly, you may receive contributions from
+outside the lab.  You should consider documenting your contribution
+policies in your `README.md` or in a `CONTRIBUTING.md` file at the
+top level of your repository.  It is good practice to make the following
+details explicit:
+
+- What license contributions should be made under
+- How to submit contributions (pull requests, branches, etc.)
+
+With most open source projects, it is assumed that contributions are made
+under the *same* license that the project is distributed under. For
+example, if you distribute your project under the `MIT` license,
+contributed code is assumed to be under that license as well. The
+[Cardioid](https://github.com/llnl/cardioid) project makes this explicit
+in its `README.md`:
+
+```
+Cardioid is distributed under the terms of the MIT license. All new
+contributions must be made under this license.
+```
+
+If you want to provide instructions to your users that they should follow
+when submitting code to your project, you can put these types of
+instructions in a top-level `CONTRIBUTING.md` file.  This file typically
```

Comment:
the contributing file is often in a .github directory. I think this enables GitHub to automatically suggest that new contributors read it first.

Reply:
Want to add a blurb about that to the pull request? I'm happy to suggest that as the place to put it.

Reply:
It'll suggest regardless of where it is, but `.github` hides it from the top level, which can be nice if you already have plenty of contributing info in your `README`.  Our `CONTRIBUTING.md` is pretty minimal -- I might move it to `.github`, as it just repeats info in the README and links to our docs.

Reply:
There are a few places where it says "top-level contributing file". Maybe just get rid of the words "top-level." The first time it's mentioned, it is made clear that the information should be documented/pointed to in the readme.

Reply:
fixed -- see update.